packages/common/src/store/pages/chat/selectors.ts

```diff
@@ -92,6 +92,11 @@ export const getChatMessages = createSelector(
   }
 )
 
+export const getHasUnreadMessages = (state: CommonState) => {
```

Comment:
fwiw I don't think this is sufficient - the user might have older unread chats from chats beyond the fetched chats page size. @michellebrier added a new endpoint for checking if there are any unreads